### DISCUSSION TITLE
Deleting selected anchors

### DISCUSSION THREAD

**Discussion by RionIwasaki** [WeirongChen/DeepEdge#8] [Q&A]
Dear Weirong,
I wonder if we can get rid of a few selected anchor points from an auto-tracked contour. For example, when an auto-tracked contour exceeds the visible range of the tongue. I've encountered various cases when the shape of the contour is very good but it just exceeds the range. In these cases, I don't want to manually correct everything. Thank you!

**Comment by WeirongChen** [comment]
You can right-click to delete an anchor point or right-click, hold and drag to erase multiple anchor points (like a pen brush eraser).   In the "Anchor" panel, you can change the "eraser width" to increase or decrease the size of pen brush eraser.  


![Screenshot 2024-04-07 195612](https://github.com/WeirongChen/DeepEdge/assets/25228432/7321d5f9-b188-49b2-a419-1137c1984df6)

**Reply by RionIwasaki** [reply]
Thank you for the quick response! Nice to know this feature is available. I will try it later.
